### ISSUE TITLE
search for geocaches in the map does not work

### ISSUE THREAD

**Issue by PontiacCZ** [2Abendsegler/GClh#422]
"Search for geocaches" in the map does not work with GC Little Helper II. Works as expected with GClh off.

**Comment by 2Abendsegler** [comment]
Hi,
I tryed it with coordinates and city and it works for me. 
![search for geocaches](https://user-images.githubusercontent.com/22332216/31035007-aed1704c-a566-11e7-9cd1-821e62bebdea.jpg)
What exactly does not work for you.
Frank

**Comment by PontiacCZ** [comment]
I was attempting to search for a town. I entered the town name and pressed Enter (tried to click the magnifier icon as well) and it did just nothing, I always waited at least half a minute.

But now.... now it works again. :) Don't know what happened. I thought some error in the code had come in the latest version but apparently it was a temporary flaw. Sorry to disturb you. :) 

Closing the ticket.
